src/controllers/DynamicCategoryFacetQueryController.ts

```diff
@@ -55,7 +56,29 @@ export class DynamicCategoryFacetQueryController {
     };
   }
 
-  public get currentValues(): IFacetRequestValue[] {
+  public executeIsolatedQuery(): Promise<IQueryResults> {
+    const query = this.facet.queryController.getLastQuery();
```

Comment:
Another question: would someone calling this method always expect to run the last query? Could there be a situation where an implementer wants finer control?